0_download_data/0.download_NF1_data.yml

```diff
@@ -0,0 +1,28 @@
+name: download-NF1-data
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.8
+  - conda-forge::scikit-image=0.19.2
+  - conda-forge::pandas=1.4.4
```

Comment:
Very interesting! 

@d33bs Can you provide what packages you think would be less likely to cause issues if the version was upgraded?

Reply:
Pandas seems to be the only discrepancy so I will upgrade the pandas in this environment since pandas in the cellprofiler pipelines environment does not come with a version specification. 

@d33bs I think that seems fair!